.env.sample

```
@@ -83,9 +83,13 @@
 ## with base64 encoded SSL certificate for DB
 # DB_CA_CERT=
 
-REDIS_ENABLED=false
+REDIS_ENABLED=true
 # redis[s]://[[username][:password]@][host][:port][/db-number]
-REDIS_URL=redis://localhost:6379
+REDIS_URL=redis://default:SLlXEEyJnRxTWGxhVqyCRYGjybuYzxdN@tramway.proxy.rlwy.net:24552
+REDIS_HOST="tramway.proxy.rlwy.net"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)